packages/protocol/contracts/thirdparty/automata-attestation/lib/EnclaveIdStruct.sol

```diff
@@ -0,0 +1,28 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
```

Comment:
for all new files, please use the exact pragma as we used in all existing solidity files.